reference-architecture/osp-cli/OSv3.yml.template

```diff
@@ -0,0 +1,36 @@
+deployment_type: openshift-enterprise
```

Comment:
Will update.  Matching the heat templates. Should it not now be changed universally to OCPv3.yml (gag)